latest/root/usr/share/container-scripts/mongodb/init-configsvr.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "${CONTAINER_SCRIPTS_PATH}/common.sh"
+
+# This is a full hostname that will be added to replica set
+# (for example, "replica-2.mongodb.myproject.svc.cluster.local")
+readonly MEMBER_HOST="$(hostname -f)"
+
+# Initializes the replica set configuration.
+#
+# Arguments:
+# - $1: host address[:port]
+#
+# Uses the following global variables:
+# - MONGODB_REPLICA_NAME
+# - MONGODB_ADMIN_PASSWORD
+function initiate() {
+  local host="$1"
+
+  local config="{_id: '${MONGODB_REPLICA_NAME}', configsvr: true, members: [{_id: 0, host: '${host}'}]}"
```

Comment:
Is this file needed? I see that `latest/root/usr/share/container-scripts/mongodb/init-replset.sh` can also set `configsvr: true`...